app/pandas.ipynb

```diff
@@ -208,18 +429,96 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "df[\"Average age\"] = df[\"Age\"].mean()\n",
+    "df"
```

Comment:
You have added a column 'Average age' twice. Remove the duplicate column to avoid redundancy.